src/bridge/cockpitinternalmetrics.c

```diff
@@ -81,8 +81,6 @@ static MetricDescription metric_descriptions[] = {
   { "disk.all.read",    "bytes", "counter", FALSE, DISK_SAMPLER },
   { "disk.all.written", "bytes", "counter", FALSE, DISK_SAMPLER },
 
-  { "network.all.rx",       "bytes", "counter", FALSE, NETWORK_SAMPLER },
```

Comment:
For better or worse, I agree. This is what we've signed up for with the compatibility between our Cockpit components. In addition if we're not using this functionality, we should write a basic unit test for it, to make sure these options are still recognized.